packages/auto-icons/package.json

```diff
@@ -1,5 +1,28 @@
 {
   "name": "@wxt-dev/auto-icons",
+  "description": "WXT module for automatically generating extension icons in different sizes",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/wxt-dev/wxt/tree/main/packages/auto-icons"
+  },
```

Comment:
This doesn't look right. See Vite's for reference. I think `packages/wxt` is also out of date.

https://github.com/vitejs/vite/blob/0f56e1724162df76fffd5508148db118767ebe32/packages/vite/package.json#L63-L67